ui/nuxeo-document-preview.js

```diff
@@ -211,8 +212,11 @@ import './viewers/nuxeo-video-viewer.js';
           // by default just use the field at xpath as a Blob
           this._blob = this.document && this._deepFind(this.document.properties, this.xpath);
         }
+        this.isBlobUpdated = !this.isBlobUpdated;
         // update our previewer
-        this._updatePreview();
+        if (this.isBlobUpdated) {
```

Comment:
Kindly re-cehck these changes.

Reply:
Checked with the necessary scenarios. Working correctly